### PR TITLE
NIAD-246 Instructions for using Fake MESH server

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,22 @@ the entire value be managed as a secured secret.
...
+
+For local test scripts see [mesh/README.md](/mesh/README.md)
+
+### OpenTest
+
+TODO
+
+### Fake MESH
+
+Clone https://github.com/mattd-kainos/fake-mesh from our fork
```
Comment (bartek-sarul) [inline README.md:210]: Why not in this repo?

File: mesh/README.md [hunk 2]
```diff
@@ -46,3 +48,33 @@ Acknowledge a message to remove it from your inbox. The message id `202006031453
...
+Use the following in your `env.sh`
+
+    export MAILBOX='anything' 
+    export MAILBOX_PASSWORD='password' 
+    export HOST='localhost:8829'
+    export SHARED_KEY='SharedKey'
+    export OPENTEST_ENDPOINT_PRIVATE_KEY='PATH_TO/fake-mesh/client.key.pem'
+    export OPENTEST_ENDPOINT_CERT='PATH_TO/fake-mesh/client.cert.pem'
+    
+The mailbox id can actually be anything you want.
```
Comment (bartek-sarul) [inline mesh/README.md:63]: Is 'mailbox id' == 'MAILBOX' ?

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

File: mesh/README.md (hunk 2 above)
Comment (mattd-kainos) [reply mesh/README.md:63]: change to MAILBOX_ID

File: README.md (hunk 1 above)
Comment (mattd-kainos) [reply README.md:210]: Different project, different tech stack, and it is a fork of someone else's repository. Better to leave separate.

**Review by grzegorz-robak — APPROVED** [review]

**Review by bartek-sarul — APPROVED** [review]

**Pull request merged**